MekHQ/src/mekhq/gui/dialog/CampaignOptionsDialog.java

```diff
@@ -3049,6 +3052,16 @@ public void actionPerformed(java.awt.event.ActionEvent evt) {
         gridBagConstraints.anchor = java.awt.GridBagConstraints.NORTHWEST;
         panNameGen.add(panRandomPortrait, gridBagConstraints);
 
+        chkAssignPortraitOnRoleChange.setText(resourceMap.getString("chkAssignPortraitOnRoleChange.text"));
+        chkAssignPortraitOnRoleChange.setToolTipText(resourceMap.getString("chkAssignPortraitOnRoleChange.toolTipText"));
+        chkAssignPortraitOnRoleChange.setName("chkAssignPortraitOnRoleChange");
+        gridBagConstraints = new java.awt.GridBagConstraints();
+        gridBagConstraints.gridx = 0;
+        gridBagConstraints.gridy = 4;
```

Comment:
Let's implement a variable to keep track of current gridy (or you could even just do a ++ on the gridy, now that I think about it).

Reply:
Since this section frequently reinitializes `gridBagConstraints` to reset the parameters as each control is different, I went with a variable to increment the `gridy` here.